libs/editor/api-v2/README.md

```diff
@@ -0,0 +1,7 @@
+# editor-api-v2
```

Comment:
what is the intention of api v2?

Reply:
Comes from https://github.com/wepublish/wepublish/pull/810, it's the version we are working on right now to use nestjs.